screen/send/details.js

```diff
@@ -1199,11 +1199,12 @@ const SendDetails = () => {
 
   const renderBitcoinTransactionInfoFields = (params) => {
     const { item, index } = params;
+    const itemAmount = item.amount ? BigNumber(item.amount).toFormat().toString() : null;
```

Comment:
The function `toFormat()` already returns a string. There is no need to call a `toString()`.